docs/client/session.md

```diff
@@ -0,0 +1,33 @@
+# Semantic conventions for session
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions to apply to client-side applications when tracking sessions.
```

Comment:
Can a session be used in a server-side opentelemetry instrumentation as well? Let's say as an end user I interact with a website, trigger some client-side API calls which get tagged with the session_id, can the same session_id be applied to API calls that might be executed on my behave in the same context.

Reply:
This was my intention with adding session as a top level namespace!